.gitignore

```diff
@@ -56,3 +56,5 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+package-lock.json
+package.json
```

Comment:
You should only edit the `simple-icons.json` and your `nhost.svg` file. Please don't any other project files in the PR.